build.sbt

```diff
@@ -2,6 +2,8 @@ name := "dfdl-tcpMessage"
 
 organization := "io.github.dfdlschemas"
 
-version := "1.1.0"
+version := "1.1.1"
+
+daffodilVersion := "4.0.0"
```

Comment:
Do we want to pin this to 4.0.0? Right now the plugins.sbt file sets sbt-daffodil to 1.+ so whenever a new sbt-daffodil plugin is released this will automatically update to the latest daffodil version.

Reply:
I think we do, because ultimately next update to the daffodil-sbt plugin the goal is to update all of these schemas that have high training/tutorial value to have build.sbt that build for several recent versions of daffodil so as to illustrate how to do that. 

I missed that you had already commented and force pushed this because I forgot the URI changes. Sorry.